readthedocs/doc_builder/constants.py

```diff
@@ -25,3 +26,16 @@
 DOCKER_OOM_EXIT_CODE = 137
 
 DOCKER_HOSTNAME_MAX_LEN = 64
+
+# Build images
+BUILD_IMAGES = {
+    'readthedocs/build:14.04': {
+        'python': {'supported_versions': [2, 2.7, 3, 3.3]},
+    },
+    'readthedocs/build:16.04': {
+        'python': {'supported_versions': [2, 2.7, 3, 3.5]},
+    },
+    'readthedocs/build:beta': {
```

Comment:
I was thinking about naming schemes myself, however I was considering how to make names more generic. I don't think it's important to have a `16.04` branch of names and a `14.04` branch of images. Perhaps we adopt semver versioning instead? I'll leave this for now, we can discuss versioning in the docker-images repo